cabot/cabotapp/models.py

```diff
@@ -334,6 +334,18 @@ class StatusCheck(PolymorphicModel):
         default=True,
         help_text='If not active, check will not be used to calculate service status and will not trigger alerts.',
     )
+    activity_counter = models.PositiveIntegerField(
+        default=0,
+    )
```

Comment:
I still like the idea of an ActivityCounter as a separate model so we don't have to have extra data for all the checks that don't require it, but don't have a super strong opinion either way.

Reply:
Yeah, I agree, however I think if I put these fields in a different model, they won't show up in the check's configuration page (correct?).  If we can figure a way around that, we could have a separate model. I can do some investigation.

Reply:
Can we keep use_activity_counter in the check and add a separate ActivityCounter model with a foreign key to status check that actually tracks the counter value? then when the endpoint is hit, find or create the corresponding activitycounter for the statuscheck and update it. I think it would be good to avoid having a field that's null for almost all statuschecks, and since the user doesn't need to input anything for the activitycounter value we don't need it in the view.

So
```
StatusCheck
   - use_activity_counter
   - etc

ActivityCounter
    - activity_count
    - status_check (ForeignKey to StatusCheck)
```

Reply:
Ok sure, i'll look into it.  Seems like a better design.